client/src/components/Layout.jsx

```diff
@@ -23,14 +23,21 @@ const theme = createMuiTheme({
       contrastText: '#fff',
     },
   },
+  
 });
 const useStyles = makeStyles(() => ({
   root: {
     textAlign: 'center',
     backgroundImage: 'linear-gradient(#d7e1fa, #bbcef9)',
-    overflowY: 'auto',
-    height: '85vh',
+    overflowY: 'scroll',
```

Comment:
On a Windows desktop, `scroll` will leave a permanent scroll bar on the screen while `auto` will add the scroll bar only if necessary.

I recommend always using `auto` over `scroll`.

Reply:
I may not be good practice but since I have a PR out I just added this change to [mine](https://github.com/COVID-19-electronic-health-system/Corona-tracker/pull/425).